src/main/webapp/pipe.js

```diff
@@ -331,105 +338,119 @@ function getLink(data, link) {
 }
 
 function generateDescription(data, task) {
-    if (data.showDescription && task.description && task.description != '') {
-        var html = ['<div class="infoPanelOuter">'];
-        html.push('<div class="infoPanel"><div class="infoPanelInner">' + task.description.replace(/\r\n/g, '<br/>') + '</div></div>');
-        html.push('</div>');
-        return html.join('');
+    if (!data.showDescription || !task.description || task.description === '') {
+        return undefined;
     }
+
+    var html = ['<div class="infoPanelOuter">'];
+    html.push('<div class="infoPanel"><div class="infoPanelInner">' + task.description.replace(/\r\n/g, '<br/>') + '</div></div>');
+    html.push('</div>');
+    return html.join('');
 }
 
 function generateTestInfo(data, task) {
```

Comment:
Please note: git shows big difference, but the changes made:
1. Inverted the condition.
2. Made indentation consistent for the lines of block, which previously was in if statement